Tests/SourceKitLSPTests/CallHierarchyTests.swift

```diff
@@ -183,4 +183,45 @@ final class CallHierarchyTests: XCTestCase {
       ]
     )
   }
+
+  func testReportSingleItemInPrepareCallHierarchy() async throws {
```

Comment:
Another test here (though not sure how easy it is with our infrastructure) would be two functions wrapped in `#if #else #endif` - they would both have the same USR but different locations.

Reply:
Discussed offline that it doesn’t make sense to test this since one file can currently on have one set of compiler arguments in sourcekit-lsp.